cypress/support/commands.js

```diff
@@ -51,9 +51,21 @@ Cypress.Commands.add('logout', () => {
   });
 });
 
-Cypress.Commands.add('toggleSwitch', name =>
-  cy.get(`[data-cy=switch-${name}]`).click()
-);
+Cypress.Commands.add('setSwitch', (name, value) => {
```

Comment:
I refactored `toggleSwitch` into `setSwitch` where you explicitly set the value of the switch so you can see exactly what it's set to in a test. `toggleSwitch` requires you to understand what the default for that switch is, which is poor.